ui/api/bookmarks.ts

```diff
@@ -62,11 +62,16 @@ export async function createBookmark({
 
 export async function getClusters(): Promise<Cluster[]> {
   const url = `${process.env.BACKEND_URL}/api/kafkas?fields%5Bkafkas%5D=name,bootstrapServers,authType`;
-  const res = await fetch(url, {
-    headers: {
-      Accept: "application/json",
-    },
-  });
-  const rawData = await res.json();
-  return Response.parse(rawData).data;
+  try {
+    const res = await fetch(url, {
+      headers: {
+        Accept: "application/json",
+      },
+    });
+    const rawData = await res.json();
+    return Response.parse(rawData).data;
+  } catch (err) {
+    console.error(err);
+    throw err;
+  }
```

Comment:
Running locally with the API at `localhost` fails without anything useful logged. Just adding this here to help debug in the future for anyone with the same issue. The solution is to use `127.0.0.1` instead.